src/util/find_uid.c

```diff
@@ -90,8 +90,15 @@ static errno_t get_uid_from_pid(const pid_t pid, uid_t *uid)
                       path);
             return EOK;
         }
+        // case of hidepid=1 mount option for /proc
+        if (error == EPERM) {
```

Comment:
Can you change the structure of the conditions to be:
```
if (error == ENOENT) {
...
} else if (error == EPERM) {
...
} 

```

This would be more consistent with the rest of the SSSD code.